polyfill/Fetch.js

```diff
@@ -40,6 +40,7 @@ class RNFetchBlobFetchPolyfill {
           promise = Blob.build(body).then((b) => {
             blobCache = b
             options.headers['Content-Type'] = 'multipart/form-data;boundary=' + b.multipartBoundary
```

Comment:
Do we need both the uppercase and lowercase version of this header?

Reply:
I'm not sure, probably not, but that's a different issue to tackle, the initial bug was sending out an invalid request since one field overwrote another